sourcegraph/sourcegraph.proto

```diff
@@ -142,7 +146,9 @@ message Repo {
 	// Fork is whether this repository is a fork.
 	bool fork = 12;
 
-	// Mirror is whether this repository is a mirror.
+	// Mirror indicates whether this repo's canonical location is on
+	// another server. Mirror repos track their upstream and are not
+	// eligible for discovery on this server.
 	bool mirror = 13;
```

Comment:
Would `sync` be a more appropriate name in that case? Then you could / enable disable sync-ing. I hope I'm not adding more confusion into this discussion.